lib/cmd/fh3/admin/environments/create.js

```diff
@@ -18,13 +19,18 @@ module.exports = {
     if(typeof params.autoDeployOnUpdate === 'undefined') {
       params.autoDeployOnUpdate = false;
     }
+   var request = {
+      _id: params.id,
+      label: params.label,
+      targets: targets,
+      autoDeployOnCreate: params.autoDeployOnCreate,
+      autoDeployOnUpdate: params.autoDeployOnUpdate,
+      noAppend: params.noAppend
+    };
 
-    return cb(null, {
-      _id : params.id, 
-      label : params.label, 
-      targets : targets, 
-      autoDeployOnCreate : params.autoDeployOnCreate,
-      autoDeployOnUpdate : params.autoDeployOnUpdate
-    });
-  }
+    if(params.noAppend){
+      console.log("Warning! noAppend flag used. Environment will not appear in enviromnent list. ");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)